econml/solutions/causal_analysis/_causal_analysis.py

```diff
@@ -715,15 +747,27 @@ def fit(self, X, y, warm_start=False):
         # assume we'll be able to train former failures this time; we'll add them back if not
```

Comment:
On line 685: you are not scaling the continuous variables when you are running the model selection for the y outcome!

Reply:
Fixed